decred_wallet/Features/Wallet Setup/RecoverExistingWalletViewController.swift

```diff
@@ -111,55 +126,50 @@ class RecoverExistingWalletViewController: WalletSetupBaseViewController, UITabl
         if self.validateSeed().valid {
             self.activateConfirmButton()
         } else {
-            self.deactivateConfirmButton()
+            self.deactivateConfirmButtonForError()
```

Comment:
> the button is not really deactivated it blocks it from proceeding and display the appropriate error message when clicked.

We have a `activateConfirmButton` function, that does not "really" activate the button either, it just changes the background color. But the name makes more sense, so does `deactivateConfirmButton`.

UPDATE:
I see that both functions now only change the button color. We can take off those functions and set the appropriate colors directly instead of calling those functions. Unless the functions are called from more than 1 place each.